lib/pgBackRest/Common/Log.pm

```diff
@@ -690,6 +693,14 @@ sub log
             if (!$bSuppressLog)
             {
                 syswrite(*STDOUT, $strMessageFormat);
+
+                # This is here for debugging purposes - it's not clear how best to make it into a switch
+                # if ($strLevel eq ASSERT || $strLevel eq ERROR)
```

Comment:
It's usually assert and error I'm after when there are not files logs.  I'm just going to leave this commented out for time being.